src/Sidenav/SideNavItemToggle.component.tsx

```diff
@@ -85,6 +86,7 @@ export const SideNavItemToggle: React.FunctionComponent<Props> = ({
 };
 
 SideNavItemToggle.defaultProps = {
+  theme: { ...Themes.defaultTheme },
```

Comment:
I think everywhere we have this, we could instead do `theme: Themes.defaultTheme`.

Reply:
Oh right, let me transform that